bin/citrea/tests/bitcoin_e2e/config/rollup.rs

```diff
@@ -30,6 +30,7 @@ pub fn default_rollup_config() -> RollupConfig {
             node_password: String::from("password"),
             network: bitcoin::Network::Regtest,
             da_private_key: None,
+            tx_backup_dir: "../../resources/bitcoin/inscription_txs".to_string(),
```

Comment:
I doubt this is the best approach for tests

Reply:
why

Reply:
discssed in dms